docs/tutorial.rst

```diff
@@ -11,19 +11,19 @@ stochastic block model (SBM), degree-corrected SBM, and random dot product graph
 
 .. toctree::
    :maxdepth: 1
-      
+
    tutorials/models/models
 
 .. _simulations_tutorials:
 
 Simulations
 ===========
-The following tutorials demonstrate how to easily sample random graphs from graph models such as the Erdos-Renyi model, 
+The following tutorials demonstrate how to easily sample random graphs from graph models such as the Erdos-Renyi model,
 stochastic block model, and random dot product graph (RDPG).
 
 .. toctree::
    :maxdepth: 1
-   
+
    tutorials/simulations/erdos_renyi
    tutorials/simulations/sbm
    tutorials/simulations/rdpg
```

Comment:
a lot of this code is  changed randomly because of spacing or something? any clue why? should be fewer arbitrary changes to make the real changes more clear

Reply:
@loftusa I am unsure. I just tried to pass the format checks and did that

Reply:
small thing, but can we make it the same as the original? no need to make formatting changes unecessarily

Reply:
I also just do not recall making these changes. I will resolve this mainly because they look the same.